datafusion/physical-plan/src/joins/hash_join.rs

```diff
@@ -293,7 +293,7 @@ impl JoinLeftData {
 ///                       │  "dimension"  │     │    "fact"     │
 ///                       └───────────────┘     └───────────────┘
 /// ```
-#[derive(Debug)]
+#[derive(Debug)] // note not Clone because of the OnceAsync
```

Comment:
Would be amazing to clarify what is OnceAsync and why clone is not fit for it?

Reply:
will try

Reply:
Here is my attempt: https://github.com/apache/datafusion/pull/13223

Reply:
Thanks, in that case we can highlight it in doc? `OnceAsync`

Reply:
Good idea -- I tried to clarify in b5ffef024